index.js

```diff
@@ -10,9 +10,61 @@ function ask(questionText) {
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  console.log(
+    "Let's play a game where you (human) pick a number, 1 - 100, and I (computer) try to guess it."
+  );
+
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n>_"
+  );
+
+  console.log("You entered: " + secretNumber);
+
+  // function random number generator
+  let max = 100;
+  let min = 1;
+  function randomInteger(min, max) {
+    let range = max - min + 1;
+    return Math.floor(Math.random() * range) + min;
+  }
+
+  // log victory and exit if yes
+  // any other response log invalid response
+
+  let guess = randomInteger(1, 100);
+  //  ask if the number is your number(random number), yes or no? and wait for answer
+  let answer = await ask(`Is your number ${guess}?\nyes or no? >_`);
+
+  // start yes, no loop
+  // if answer === yes then log out. if no then hi low
+  if (answer.toLowerCase().trim() === "yes") {
+    //if yes log victory and exit
+    console.log("A.I. is VICTORIOUS!!!");
+    process.exit();
+  } else {
+    //if answer is no ask if number is high or low and await answer
+    answer.toLowerCase() === "no";
+    answer = await ask("Is your number higher or lower? \nh or l? >_");
```

Comment:
I see below you commented out some await asks that assign the High/Low response to a variable other than "answer", definitely a good idea since having them reassign the "answer" variable can get messy.